src/modelfree/score_agent.py

```diff
@@ -50,6 +57,62 @@ def get_empirical_score(_run, env, agents, episodes, render=False):
     return result
 
 
+def _clean_video_directory_structure(observer_obj):
+    """
+    A simple utility method to take saved videos within a Sacred run structure and clean
+    up the file pathways, so that all videos are organized under a "videos" directory
+
+    :param observer_obj: A Sacred FileStorageObserver object
+    :return: None
+    """
+    basedir = observer_obj.dir
+    video_files = glob.glob("{}/*.mp4".format(basedir))
+    metadata_files = glob.glob("{}/*metadata.json".format(basedir))
+    if len(video_files) == 0:
+        return
+
+    new_video_dir = os.path.join(basedir, "videos")
+    os.mkdir(new_video_dir)
+    new_video_metadata_dir = os.path.join(new_video_dir, "metadata")
+    os.mkdir(new_video_metadata_dir)
```

Comment:
You could use `os.makedirs(new_video_metadata_dir)` to do this and the previous `os.mkdir` in one fell swoop, but I'm not sure that's actually better.

Reply:
Yeah, I think I like being a bit more procedural about it, even if it technically could be condensed